lib/helpers/isAxiosError.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/**
+ * Determines whether the payload is an error thrown by Axios
+ *
+ * @param {*} payload The value to test
+ * @returns {boolean} True if the payload is an error thrown by Axios, otherwise false
+ */
+module.exports = function isAxiosError(payload) {
+  return (typeof payload === 'object') && (payload.isAxiosError === true);
```

Comment:
`typeof null === 'object'` => `true`

Should be changed to:

`({}).toString.call(payload) === '[obejct Object]'`

Reply:
Is it necessary to make a strict inspection here?  @jasonsaayman @JasonHK

Reply:
Oops, I totally forgot about this. But because this PR had already been merged, I will open another PR to fix this.

Reply:
OK~

Reply:
I have submitted the relevant PR: https://github.com/axios/axios/pull/3546